### PR TITLE
Add contributor covenant v1.3 as Code of Conduct

### LINKED ISSUE #13: Add CoC
This is probably better done early and amended than done late and fought over.

### PR DESCRIPTION
**Pull request by danielcompton**
The contributor covenant is pretty widely used across the open source
world, it seems like as good a choice as any.

Fixes #13

### CODE REVIEW

**Comment by arrdem** [comment]
@tbarn anything else we should look at?

@danielcompton needs a rebase

**Comment by arrdem** [comment]
Also probably warrants a mention in `/readme.md`

**Comment by danielcompton** [comment]
Also need an email to report to.

**Comment by tbarn** [comment]
So clojarr is actually happening.... Okay.

It would be nice to also make a CONTRIBUTING.md for your contribution stuff. That's what Github uses to show up on the top of PRs.

**Comment by danielcompton** [comment]
Done, just need an email to report to @arrdem.

**Comment by arrdem** [comment]
Ok, should be all set.

File: README.md [hunk 1]
```diff
@@ -48,6 +48,12 @@ This is not to say that contributions, issues and soforth are not
 welcome, merely to clearly state my expectations for the project and
 its management.
 
+## Contributing
+
+This project adheres to the Contributor Covenant [code of conduct](CODE_OF_CONDUCT.md).
+By participating, you are expected to uphold this code.
+Please report unacceptable behavior to [?@arrdem.com](mailto:?@arrdem.com).
```
Comment (danielcompton) [inline README.md]: me@arrdem

**Pull request merged**